File: Programmers/Lv2/올바른 괄호.py
def solution(s):
    answer = True

    stack = []

    for w in s:
        if w == '(':
            stack.append(w)
        else:
            if len(stack) <= 0:
                return False
            stack.pop()

    if len(stack) == 0:
        return True
    else:
        return False

File: Programmers/Lv2/올바른 괄호.js
function solution(s) {
  var answer = true;

  var stack = new Array();

  for (w of s) {
    if (w === "(") {
      stack.push("(");
    } else {
      if (stack.length == 0) {
        return false;
      }
      stack.pop();
    }
  }

  if (stack.length > 0) {
    return false;
  }

  return answer;
}
